site/src/examples/stepper/Basic.tsx

```diff
@@ -0,0 +1,14 @@
+import { StackLayout } from "@salt-ds/core";
+import { Step, Stepper } from "@salt-ds/lab";
+
+export const Basic = () => {
+  return (
+    <StackLayout style={{ height: "40px", width: "100%" }}>
+      <Stepper>
+        <Step label="Step 1" stage="completed" />
```

Comment:
Stick with the common naming convention, `ParentComponentChildComponent` if possible

Reply:
Renamed Steper to SteppedTracker, gonna keep Step though.

Reply:
Keep open so other reviews can comment

Reply:
@origami-z. Yes it was considered at the time. When the collaboration started they wanted us to align to Story requirements from [Progress stepper ](https://storyds-catalog.apps.test.na-4z.gap.jpmchase.net/#/components/progress-stepper)and [Wizard](https://storyds-catalog.apps.test.na-4z.gap.jpmchase.net/#/components/wizard). However, Salt scales systematically across densities so we worked to find a middle point. One thing to highlight is that they did not use the density foundation concept at the time. See [icon parity](https://www.figma.com/design/r5jCbd7iFfiSGc9wvSfRWM/Stepped-Tracker?node-id=982-7043&t=Qxx79oNlzvKzZCwG-1) work at the time. 

That was one of the reasons the icons size changed to icon-size (150%) for steps and icon-size for sub-steps. So, they have close parity in TD and LD. Also, as a consequence of that collaboration icon size scaling in Salt evoloved offering the 100%, 200%, 300% and %400s + middle points scaling for icons. However, for stepped tracker scaling above the 200% can have a negative impact in the visual balance. 

I don't know if requirements have changed over time. LMK if that answers your question, TY!